cms/envs/common.py

```diff
@@ -732,6 +732,16 @@
             continue
     INSTALLED_APPS += (app_name,)
 
+############## Milestones ##############
+# Turn off milestones app by default
+FEATURES['MILESTONES_APP'] = False
+if FEATURES['MILESTONES_APP']:
+    INSTALLED_APPS += ('milestones',)
```

Comment:
Is this configuration necessary? This file imports the LMS common.py.

import lms.envs.common